### ISSUE TITLE
Installation Error 

### ISSUE THREAD

**Issue by vermanavneet** [bklynhlth/openwillis#84]
I am using python3.10
```
(openwillis_env) Navneets-MacBook-Pro:openwillis denny$ python --version 
Python 3.10.13

```

### Error 

```
INFO: pip is still looking at multiple versions of openwillis to determine which version is compatible with other requirements. This could take a while.
  Using cached openwillis-0.4.3.tar.gz (25 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [16 lines of output]
      /Users/denny/anaconda3/envs/openwillis_env/lib/python3.10/site-packages/setuptools/dist.py:498: SetuptoolsDeprecationWarning: Invalid dash-separated options
      !!
      
              ********************************************************************************
              Usage of dash-separated 'description-file' will not be supported in future
              versions. Please use the underscore name 'description_file' instead.
      
              This deprecation is overdue, please update your project and remove deprecated
              calls to avoid build errors in the future.
      
              See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
              ********************************************************************************
      
      !!
        opt = self.warn_dash_deprecation(opt, section)
      error in openwillis setup command: 'python_requires' must be a string containing valid version specifiers; Invalid specifier: '>=3.6.*'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by vermanavneet** [comment]
@anzarabbas @vjbytes102 @GeorgeEfstathiadis Please take a look at this urgently. Thank You.

**Comment by vjbytes102** [comment]
@vermanavneet, multiple possibilities can be considered here:
 - PyPI might have cached information about the old version of openwillis in your machine or
 - your machine may have older versions for some supporting libraries that force to install the older version of openwillis
 
did you create a fresh openwillis_env virtual environment ? if yes give it a try with pip install openwillis==1.6

**Comment by vermanavneet** [comment]
@vjbytes102 Yes, I created a fresh openwillis_env virtual environment. I  tried  with pip install openwillis==1.6 but got following error
```
(openwillis_env) Navneets-MacBook-Pro:openwillis denny$ pip install openwillis==1.6
Collecting openwillis==1.6
  Using cached openwillis-1.6.tar.gz (39 kB)
  Preparing metadata (setup.py) ... done
Collecting tensorflow==2.12.0 (from openwillis==1.6)
  Using cached tensorflow-2.12.0-cp310-cp310-macosx_10_15_x86_64.whl (230.1 MB)
INFO: pip is looking at multiple versions of openwillis to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement mediapipe==0.10.1 (from openwillis) (from versions: 0.8.9.1, 0.8.10, 0.8.10.1, 0.8.11, 0.9.0, 0.9.0.1, 0.9.1.0)
ERROR: No matching distribution found for mediapipe==0.10.1
(openwillis_env) Navneets-MacBook-Pro:openwillis denny$ 


```

**Comment by vjbytes102** [comment]
@vermanavneet are you installing it on M1/ or M2 chip?

**Comment by vermanavneet** [comment]
@vjbytes102 On Intel Chip. Though got that thing resolved but now stuck on this issue

```
(openwillis_env) Navneets-MacBook-Pro:openwillis denny$ pip3 install vosk==0.3.45
ERROR: Could not find a version that satisfies the requirement vosk==0.3.45 (from versions: 0.3.21, 0.3.29, 0.3.30, 0.3.31, 0.3.32, 0.3.38, 0.3.40, 0.3.41, 0.3.42, 0.3.43, 0.3.44)
ERROR: No matching distribution found for vosk==0.3.45

```

**Comment by vjbytes102** [comment]
All installation-related issues have been addressed and fixed in the latest version (v2.1.1).
